lampod/src/lib.rs

```diff
@@ -106,6 +113,45 @@ impl LampoDaemon {
         &self.conf
     }
 
+    pub fn liquidity(&self) -> Arc<LampoLiquidityManager> {
+        self.liquidity.clone().unwrap()
+    }
+
+    // This should not be initiated when we open our node only when we
+    // provide some args inside cli or in lampo.conf
+    fn init_lampo_liquidity(
```

Comment:
follow the name convention of other methods

Reply:
Thanks